src/tests/src/solver/optimisation/adequacy_patch.cpp

```diff
@@ -0,0 +1,160 @@
+#define BOOST_TEST_MODULE test adequacy patch functions
+
+#define WIN32_LEAN_AND_MEAN
+
+#include <boost/test/included/unit_test.hpp>
+
+#include "adequacy_patch.h"
+
+#include <vector>
+#include <tuple>
+
+static double origineExtremite = -1;
+static double extremiteOrigine = 5;
+
+using namespace Antares::Data::AdequacyPatch;
+
+// NOTE
+// Xmax limits the flux origin -> extremity (direct)
+// -Xmin limits the flux extremity -> origin (indirect)
+
+std::pair<double, double> one_timestep(AdequacyPatchMode originType,
```

Comment:
**one_timestep** function should be renamed **setNTCboundsForOneTimeStep** I suppose

Reply:
Done